src/main/scala/scala/compat/java8/collectionImpl/Stepper.scala

```diff
@@ -0,0 +1,545 @@
+package scala.compat.java8.collectionImpl
+
+import java.util.Spliterator
+
+/** A Stepper is a specialized collection that can step through its
```

Comment:
The Spliterator interface is low-performance compared to the next/hasNext interface of Iterators.  Since Stepper provides the latter, it can be valuable to have access to it.